docs/configuration/translation-strings.md

```diff
@@ -0,0 +1,65 @@
+# Translation Strings
+
+> Translation Strings are multilingual key-value pairs that you can use throughout the app. They enable you to translate
+> things like dropdown options, placeholder text, field notes, and more.
+
+[[toc]]
+
+## How It Works
+
+<video autoplay muted loop controls>
+<source src="https://cdn.directus.io/docs/v9/configuration/translation-strings/translation-strings-20220615A/translation-strings-20220615A.mp4" type="video/mp4" />
+</video>
+
+When creating a Field, you may want to add a note or other description along with it to clarify its context or purpose.
+While you always have the option to type in a static value, Translation Strings let you group together translations
+which display dynamically, whenever the app language is changed. There are two locations where the language could be
+set. First, administrators can set the Project's default language under **Settings > Project Settings**. Second, Users
+can choose their own personal [language preference](/app/user-directory/#user-preferences) from the user details page.
+
+## Create and Manage a Translation String
+
+<video autoplay muted loop controls>
+<source src="https://cdn.directus.io/docs/v9/configuration/translation-strings/translation-strings-20220616A/create-and-manage-a-trasnlation-string-20220616A.mp4" type="video/mp4" />
+</video>
+
+To create and manage a Translation String, follow these steps.
+
+1. Navigate to **Settings > Translation Strings**.
+2. Click <span mi btn>add</span> in the page header and a drawer will open.
+3. Add a key and click **"Create New"** and another drawer will open.
+4. Select the language and type in the corresponding translation.
+5. Click <span mi btn>check</span> to confirm and add the translation.
+6. Repeat steps 3-5 as desired.
+7. Optional: Click <span mi icon>close</span> to delete a translation.
+8. Click <span mi btn>check</span> to confirm and save the Translation Strings.
+
+## Assign and Remove a Translation String
+
+<video autoplay muted loop controls>
+<source src="https://cdn.directus.io/docs/v9/configuration/translation-strings/translation-strings-20220616A/assign-and-remove-translation-string-20220616A.mp4" type="video/mp4" />
+</video>
+
+Throughout the **Data Model** as well as **Presets and Bookmarks** sections in the Settings Module, you will notice
```

Comment:
To make this a bit more future proof, let's lot list those two sections. Just say "Within Settings". Also, Settings is global enough to not always include `... Module`.

Reply:
Done. *(Although, I personally think this works better at this present moment, where the configuration docs are so lacking. Helps points users in the right direction)*

Also:
- I have to update this article anyways as soon as Azzy slashes the drag-n-drop.
- The team consistently tags me for updates on the app.

But I'm happy when you're happy.